graphics/MoltenVK/Portfile

```
@@ -1,140 +1,71 @@
-distfiles           ${distfiles}:${name}
+homepage            https://github.com/KhronosGroup/MoltenVK
 
-foreach {sub_author sub_project sub_commit sub_dest} ${submodules} {
-    master_sites-append https://github.com/${sub_author}/${sub_project}/archive/${sub_commit}.tar.gz?dummy=:${sub_project}
-    distfiles-append    ${sub_project}-${sub_commit}.tar.gz:${sub_project}
-}
+distname            vulkansdk-macos-${sdkversion}
+use_dmg             yes
 
-post-extract {
-    ln -s {*}[glob -directory ${workpath} -tails ${github.author}-${github.project}-*] ${worksrcpath}
-    set bad_submodule no
-    foreach {sub_author sub_project sub_commit sub_dest} ${submodules} {
-        set commit_file ${worksrcpath}/ExternalRevisions/${sub_project}_repo_revision
-        if {[file exists ${commit_file}]} {
-            set commit_re {^(\S+)}
-        } else {
-            set commit_file ${worksrcpath}/External/glslang/known_good.json
-            set commit_re "\"subrepo\" : \"${sub_author}/${sub_project}\".*?\"commit\" : \"(\[^\"\]+)\""
-        }
-        set fp [open ${commit_file} r]
-        set commit_file_data [read ${fp}]
-        close ${fp}
-        regexp ${commit_re} ${commit_file_data} -> commit
-        if {${commit} ne ${sub_commit}} {
-            set bad_submodule yes
-            set submodules [string map "${sub_commit} ${commit}" ${submodules}]
-        }
-        file mkdir [file dirname ${worksrcpath}/External/${sub_dest}]
-        ln -s {*}[glob ${workpath}/${sub_project}-*] ${worksrcpath}/External/${sub_dest}
-    }
-    if {${bad_submodule}} {
-        ui_error "The submodules should be:\n${submodules}"
-        return -code error "incorrect submodule commit"
-    }
-}
-
-depends_build-append \
-                    path:bin/cmake:cmake
+master_sites        https://sdk.lunarg.com/sdk/download/${sdkversion}/mac/
 
-build.pre_args      -derivedDataPath ./DerivedData
+checksums           sha256  50accc4523bd097e4c3b7c34077408328abaf880c0501f3dc37cf2457c19bb5c \
+                    rmd160  ccf1a4a8e4a091f26bd682b94ef1009ac2da31f9 \
+                    size    127857139
 
-compiler.cxx_standard 2011
+depends_build       port:p7zip
+depends_skip_archcheck p7zip
 
-xcode.configuration Release
-xcode.scheme        "MoltenVK Package (macOS only)"
+use_configure       no
 
-pre-build {
-    set build_dir ${worksrcpath}/External/glslang/build
-    file mkdir ${build_dir}
-    system -W ${build_dir} "${prefix}/bin/cmake -DCMAKE_BUILD_TYPE=MacPorts -DCMAKE_INSTALL_PREFIX=install -DCMAKE_INSTALL_NAME_DIR=${prefix}/lib -DCMAKE_C_COMPILER=\"${configure.cc}\" -DCMAKE_CXX_COMPILER=\"${configure.cxx}\" -DCMAKE_OSX_ARCHITECTURES=\"[join [get_canonical_archs] \;]\" -DCMAKE_VERBOSE_MAKEFILE=ON .. && make -j${build.jobs} VERBOSE=ON && make install/fast"
+build {
+    # bypass the installer that requires macOS 10.13
+    system -W /bin "${prefix}/bin/7z x -aoa ${worksrcpath}/InstallVulkan.app/Contents/Resources/installer.dat -o${workpath}/VulkanSDK"
```

Why does the working directory need to be set to /bin?
```suggestion
    system "${prefix}/bin/7z x -aoa ${worksrcpath}/InstallVulkan.app/Contents/Resources/installer.dat -o${workpath}/VulkanSDK"
```